mautic/mautic
# M3 add new cache bundle 2

Original and closed PR is here https://github.com/mautic/mautic/pull/8817

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | feature
| Bug fix?                               | no
| New feature?                           | yes
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | yes
| Related user documentation PR URL      | /
| Related developer documentation PR URL | https://github.com/mautic/developer-documentation/pull/179
| Issue(s) addressed                     | 

#### Description:

This bundle enables the developer to use common Symfony caching mechanism with tag-aware adapters. It implements FS, Memcache and Redis adapters.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Set up an adapter as described in documentation
3. Use the cache as described in documentation
4. Verify items are created (e.g. redis records)
5. Verify items expire
6. Clear cache and verify items are purged
